cypress/integration/ui/view_dashboard/index.js

```diff
@@ -0,0 +1,93 @@
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps'
+
+const antenatalCareDashboardRoute = '#/nghVC4wtyzi'
+const immunizationDashboardRoute = '#/TAMlzYkstb7'
+
+Given('I open the Antenatal Care dashboard', () => {
+    cy.visit('/')
+    cy.get('[data-test="dhis2-uicore-chip"]')
+        .contains('Antenatal Care')
+        .click()
+})
+
+Then('the Antenatal Care dashboard displays in view mode', () => {
+    cy.location().should(loc => {
+        expect(loc.hash).to.equal(antenatalCareDashboardRoute)
+    })
+
+    cy.get('[data-test="dhis2-dashboard-view-dashboard-title"]')
```

Comment:
This is currently up for discussion in our coming frontend meeting, but I would argue for using something like `getBySel` ([source](https://docs.cypress.io/guides/references/best-practices.html#Real-World-Example)) to shorthand the `data-test` and/or the `dhis2-dashboard` prefix to avoid repetition.

Reply:
Will update my tests according to the decision made in the upcoming meeting